src/main/java/blusunrize/immersiveengineering/ImmersiveEngineering.java

```diff
@@ -96,7 +97,7 @@ public ImmersiveEngineering()
 		FMLJavaModLoadingContext.get().getModEventBus().addListener(this::wrongSignature);
 		MinecraftForge.EVENT_BUS.addListener(this::serverStarting);
 		MinecraftForge.EVENT_BUS.addListener(this::registerCommands);
-		MinecraftForge.EVENT_BUS.addListener(this::addReloadListeners);
+		MinecraftForge.EVENT_BUS.addListener(EventPriority.LOWEST, this::addReloadListeners);
```

Comment:
Did you verify that moving everything to lowest doesn't mess with recycling? Specifically, have you tried on servers?

Reply:
I did some rudimentary testing on a server in deobf.
The recycling recipes were still there and usable.

Anything in particular you want looked at?

Reply:
From what I remember when I looked into it, I believe you need to move to 2 even listeners, one for adding recipes, and one for caching them, that would be the safest option anyway, lets mods make changes in `LOW`